File: CSharp/GenericTree/TreeProgram.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GenericTree
{
    class TreeProgram
    {
        static void Main(string[] args)
        {

            Tree<int> myTree = new Tree<int>();

            for(int i=0; i<20; i++)
            {
                Random rand = new Random(i*DateTime.Now.Millisecond);
                myTree.InsertData(rand.Next(0, 99));
            }

            Console.WriteLine("Printing Tree in PreOrder");
            myTree.PreOrderPrint();
            Console.WriteLine("\n");
            Console.WriteLine("Printing Tree in InOrder");
            myTree.InOrderPrint();
            Console.WriteLine("\n");
            Console.WriteLine("Printing Tree in PostOrder");
            myTree.PostOrderPrint();

            Console.ReadKey();
        }
    }
}

File: Java/src/IntegerTree/TreeProgram.java
package IntegerTree;

import java.util.Random;

public class TreeProgram {

	public static void main(String[] args) {
		Tree myTree = new Tree();
		
		

        for(int i=0; i<20; i++)
        {
        	
            Random rand = new Random(i * System.currentTimeMillis());
            myTree.InsertData(rand.nextInt(99));
        }
        System.out.println();
        System.out.println("Printing Tree in PreOrder");
        myTree.PreOrderPrint();
        System.out.println("\n");
        System.out.println("Printing Tree in InOrder");
        myTree.InOrderPrint();
        System.out.println("\n");
        System.out.println("Printing Tree in PostOrder");
        myTree.PostOrderPrint();

	}

}
